src/services/balancer/pools/exits/handlers/exact-out-exit.handler.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { getBalancer } from '@/dependencies/balancer-sdk';
+import {
+  defaultExactInExit as defaultExactInExit,
+  initBalancerWithDefaultMocks,
+} from '@/dependencies/balancer-sdk.mocks';
+import { GasPriceService } from '@/services/gas-price/gas-price.service';
+import { Pool } from '@/services/pool/types';
+import { aWeightedPool } from '@/__mocks__/weighted-pool';
+import { buildExitParams } from '@tests/unit/builders/join-exit.builders';
+import {
+  defaultGasLimit,
+  defaultTransactionResponse,
+} from '@tests/unit/builders/signer';
+import { DeepMockProxy, mockDeep } from 'vitest-mock-extended';
+import { ref } from 'vue';
+import { ExactOutExitHandler } from './exact-out-exit.handler';
+
+initBalancerWithDefaultMocks();
+
+const gasPriceServiceMock: DeepMockProxy<GasPriceService> =
+  mockDeep<GasPriceService>();
+
+async function mountExactOutExitHandler(pool: Pool) {
+  return new ExactOutExitHandler(ref(pool), getBalancer(), gasPriceServiceMock);
+}
+
+const exitParams = buildExitParams({
+  bptIn: '0.00000000000000001',
```

Comment:
For exact out handler it's the `amountsOut` that are critical in constructing the tx, bptIn is irrelevant.

Reply:
Thanks. I will improve this tests with this feedback in a future PR.